ups/verify.table

```diff
@@ -1,4 +1,8 @@
-setupRequired(sconsUtils)
+setupRequired(astropy)
+setupRequired(pyyaml)
+setupRequired(requests)
+setupRequired(numpy)
+setupRequired(base)
 setupRequired(utils)
 setupRequired(pex_exceptions)
```

Comment:
You could probably remove this dependency by noting that `lsst.pex.exceptions.NotFoundError` looks like a `LookupError` in Python land.

Reply:
Can I just switch it without introducing a breaking API change?

Reply:
There is no API change is there? I think you only use NotFoundError when you are using getPackageDir. Changing those to LookupError should be fine shouldn't it?

Reply:
ah yes. perfect